src/main/java/javapoz24/team3/rental/application/config/MockData.java

```diff
@@ -72,11 +72,11 @@ public void GenerateMockData() {
         );
 
         CompanyBranch branch1 = new CompanyBranch(addressOfBranch1, rental, new HashSet<>(), new HashSet<>(),
```

Comment:
Jeżeli masz aż tyle nowych list to można od strony CompanyBranch od razu je ustawić.
private List<x> list = new ArrayList();

Reply:
Z setem to samo

Reply:
Sugerowane zmiany w nowym commicie. Musiałem dodać nowy konstruktor - bez list i setów. Faktycznie wygląda to lepiej.